foreman_remote_execution.gemspec

```diff
@@ -15,12 +15,8 @@ Gem::Specification.new do |s|
   s.description = 'A plugin bringing remote execution to the Foreman, completing the config ' +
                   'management functionality with remote management functionality.'
 
-  s.files = `git ls-files`.split("\n").reject do |file|
-    file.start_with?('scripts')
-  end
-
-  s.test_files =       `git ls-files test`.split("\n")
-  s.extra_rdoc_files = Dir['README*', 'LICENSE']
+  s.files = Dir['{app,config,db,extra,lib,locale,webpack}/**/*'] + ['LICENSE', 'Rakefile', 'README.md', 'package.json']
```

Comment:
Note: by not listing this from git, it can silently include files not tracked in git. Since it uses the release flow, this is probably not a big deal but I want to explicitly point it out.